<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPlainTextEdit" name="txtupdate">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>280</y>
      <width>445</width>
      <height>217</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>104</y>
      <width>171</width>
      <height>85</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Visualise</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="reconstructfrm2d_btn">
       <property name="text">
        <string>3D from Two views</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="reconstructfromNd_btn">
       <property name="text">
        <string>3D from Multiple View</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>212</y>
      <width>253</width>
      <height>52</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Compute</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sfm_btn">
       <property name="text">
        <string>Compute SfM</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>12</y>
      <width>225</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Load Data</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="load_IMG_btn">
         <property name="text">
          <string>Load Images </string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>100</y>
      <width>285</width>
      <height>89</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Load Calibration Matrix</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="load_calibration_btn">
       <property name="text">
        <string>Load Camera intrinsic parameters (xml)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="set_camera_intrinsic_btn">
       <property name="text">
        <string>Set Camera Intrinsic</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>sfm_btn</zorder>
   <zorder>txtupdate</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>set_camera_intrinsic_btn</zorder>
   <zorder>load_calibration_btn</zorder>
   <zorder>label_4</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>532</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionLoad_Image"/>
    <addaction name="actionLoad_Calibration_data"/>
    <addaction name="actExit"/>
   </widget>
   <widget class="QMenu" name="menuCalibration">
    <property name="title">
     <string>Set Camera Intrinsic</string>
    </property>
    <addaction name="actionCreate_Calibration_Matrix"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuCalibration"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionLoad_Image">
   <property name="text">
    <string>Load Images ...</string>
   </property>
  </action>
  <action name="actionLoad_Calibration_data">
   <property name="text">
    <string>Load Camera Parameter Matrix</string>
   </property>
  </action>
  <action name="actionCreate_Calibration_Matrix">
   <property name="text">
    <string>Create Camera Matrix ...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
